fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/CachedStatisticStorage.java

```diff
@@ -59,7 +59,12 @@ public class CachedStatisticStorage implements StatisticStorage {
             .buildAsync(new ColumnHistogramStatsCacheLoader());
 
     @Override
-    public TableStatistic getTableStatistic(Long tableId, Long partitionId) {
+    public TableStatistic getTableStatistic(Table table, Long tableId, Long partitionId) {
+        // get Statistics Table column info, just return default column statistics
+        if (StatisticUtils.statisticTableBlackListCheck(table.getId())) {
```

Comment:
If you just want to use table.getId(), the secend paramter is enough?

Reply:
oh!!! your are right, it's my mistakes, think you very much T-T